db/migrate/20210920171145_create_subscriptions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateSubscriptions < ActiveRecord::Migration[6.1]
+  def change
+    create_table :subscriptions do |t|
+      t.string :title
+      t.float :price
+      t.integer :status
```

Comment:
Currently set up as an integer so that it could be used as an enum, but may change to string for simplicity.